### DISCUSSION TITLE
Question (not an issue)

### DISCUSSION THREAD

**Discussion by Npa972** [DeaDBeeF-Player/deadbeef#2604] [Q&A]
### Steps to reproduce
Deadbeef plays very well MP3 and AAC (deadbeef devel)

### What's going on? Describe the problem in as much detail as possible.
mpg123, libmad or faad2 are not installed on my system.
So why are these packages mentioned as dependencies ?

### Information about the software:

Deadbeef version:
OS: archlinux ; DDBF static devel (14/05/2021)

**Comment by Oleksiy-Yakovenko** [comment]
Static build includes most dependencies, therefore you don't need to install them.
They're necessary if you're compiling deadbeef yourself, or if you're using a different build, which uses system-provided dependencies.
